### PR TITLE
Set client info

### PR DESCRIPTION
**Pull request by hone1er**
### What changes were made?

- Merged current feature-application-flow(with "client" login)
- Set contractOwnerName to the clientUserName instead of current userState.userName

### Is there any background info about the change?

- Allows admin to select a client and create a SOW for them
- Sets contractOwnerName to the clientUserName
- All other data for the bounty/contract is unchanged
- will need to add a filter to the client dashboard to only display jobs from the currently selected client (this would be a temporary filter until clients have their own profiles attached to unique wallets)
- still need to add the ability to fetch the list of clients on the login page & save new clients to the DB(currently hard coded a list of clients for front end testing)

### Are there any state changes? Mention key ones (if any)

na

### Shortcut Link

https://app.shortcut.com/twali/story/1447/admin-login-to-manage-clients

### CODE REVIEW

File: components/Marketplace/BountyCard.tsx [hunk 1]
```diff
@@ -54,9 +47,12 @@ interface BountyCard {
  * @param {string} contract_id ID of the current bounty
  * @param {boolean} isMarketplace whether the bounty is being displayed in the marketplace
 <<<<<<< HEAD
+=======
```
Comment (nagmak) [inline components/Marketplace/BountyCard.tsx:50]: hey looks like some merge lines were left here @hone1er

Comment (hone1er) [reply components/Marketplace/BountyCard.tsx]: Whoops! Good catch! Got that taken out. Also added a check for validity on expertise questions in whitelist application and hid the post a job button for experts in the dashboard

Comment (nagmak) [reply components/Marketplace/BountyCard.tsx]: Okay perfect!

**Review by nagmak — APPROVED** [review]
Looks good!! 🔥🔥🔥

**Comment by nagmak** [comment]
Merging this!

**Pull request merged**